source/common/common/utility.cc

```diff
@@ -196,4 +196,11 @@ const std::string& StringUtil::nonEmptyStringOrDefault(const std::string& s,
   return s.empty() ? default_value : s;
 }
 
+std::string StringUtil::toUpper(const std::string& s) {
```

Comment:
This actually is leftover from the virtual cluster priorities and the enum conversion there. I'll remote it from this PR.